pages/account/statistics.js

```diff
@@ -58,16 +60,34 @@ export async function getServerSideProps(context) {
       }
     );
     data = await res.json();
+
+    let profileSections = [
+      "bio",
+      "links",
+      "milestones",
+      "tags",
+      "socials",
+      "location"
+    ]
```

Comment:
This is not following the projects coding standards, VScode should fix this for you though

```suggestion
      "socials",
      "location",
    ];
```

Reply:
Is there any place else it needs to be fixed?

Reply:
Yes there are a few places, but I can fix via a temporary branch